### PR TITLE
Bump go-auth0 to v0.11.0 

### PR DESCRIPTION
**Pull request by sergiught**
Bumps [github.com/auth0/go-auth0](https://github.com/auth0/go-auth0) from 0.10.1 to 0.11.0.
<details>
<summary>Release notes</summary>
<p><em>Sourced from <a href="https://github.com/auth0/go-auth0/releases">github.com/auth0/go-auth0's releases</a>.</em></p>
<blockquote>
<h2>v0.11.0</h2>
<h3>⚠️ Notice of Breaking Changes</h3>
<p>Please note that this version introduces many minor breaking changes. The intention with this release was to capture as many breaking changes into a single release to minimize future disruption.</p>
<p>These changes were introduced for three primary reasons:</p>
<ul>
<li>Decoupling from <a href="https://github.com/auth0/terraform-provider-auth0">Auth0 Terraform Provider</a> implementation</li>
<li>More precise typing of struct properties for better DX</li>
<li>Enabling Auth0 Terraform provider to explicitly set empty values (see: <a href="https://github-redirect.dependabot.com/auth0/terraform-provider-auth0/issues/14">related Github issue</a>)</li>
</ul>
<p>Review the complete list below before upgrading.</p>
<h3>Changed</h3>
<ul>
<li><code>Action</code> struct fields
<ul>
<li><code>SupportedTriggers</code> to <code>[]ActionTrigger</code> type</li>
<li><code>Dependencies</code> to <code> *[]ActionDependency</code> type</li>
<li><code>Secrets</code> to <code>*[]ActionSecret</code> type</li>
</ul>
</li>
<li><code>Client</code> struct fields
<ul>
<li><code>Callbacks</code> to <code>*[]string</code> type</li>
<li><code>AllowedOrigins</code> to <code>*[]string</code> type</li>
<li><code>WebOrigins</code> to <code>*[]string</code> type</li>
<li><code>ClientAliases</code> to <code>*[]string</code> type</li>
<li><code>AllowedClients</code> to <code>*[]string</code> type</li>
<li><code>AllowedLogoutURLs</code> to <code>*[]string</code> type</li>
<li><code>EncryptionKey</code> to <code>*map[string]string</code> type</li>
<li><code>GrantTypes</code> to <code>*[]string</code> type</li>
<li><code>ClientMetadata</code> to <code>*map[string]string</code> type</li>
<li><code>Mobile</code> to dedicated <code>*ClientMobile</code> type</li>
<li><code>Scopes</code> to <code>*map[string]string</code> type</li>
</ul>
</li>
<li><code>ClientNativeSocialLogin</code> struct
<ul>
<li><code>Apple</code> to dedicated <code>*ClientNativeSocialLoginSupportEnabled</code> type</li>
<li><code>Facebook</code> to dedicated <code>*ClientNativeSocialLoginSupportEnabled</code> type</li>
</ul>
</li>
<li><code>ClientGrant</code> struct field
<ul>
<li><code>Scope</code> to <code>[]string</code> type</li>
</ul>
</li>
<li><code>Connection</code> struct fields
<ul>
<li><code>EnabledClients</code> to <code>*[]string</code> type</li>
<li><code>Realms</code> to <code>*[]string</code> type</li>
<li><code>Metadata</code> to <code>*map[string]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptions</code> struct fields</li>
<li><code>CustomScripts</code> to <code>*map[string]string</code> type</li>
<li><code>Configuration</code> to <code>*map[string]string</code> type</li>
<li><code>ConnectionOptionsGoogleOAuth2</code> struct field
<ul>
<li><code>AllowedAudiences</code> to <code>*[]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptionsOIDC</code> struct field
<ul>
<li><code>DomainAliases</code> to <code>*[]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptionsOAuth2</code> struct field
<ul>
<li><code>Scripts</code> to <code>*map[string]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptionsAD</code> struct fields
<ul>
<li><code>DomainAliases</code> to <code>*[]string</code> type</li>
</ul>
</li>
</ul>

</blockquote>
<p>... (truncated)</p>
</details>
<details>
<summary>Changelog</summary>
<p><em>Sourced from <a href="https://github.com/auth0/go-auth0/blob/main/CHANGELOG.md">github.com/auth0/go-auth0's changelog</a>.</em></p>
<blockquote>
<h2><a href="https://github.com/auth0/go-auth0/tree/v0.11.0">v0.11.0</a> (2022-10-07)</h2>
<p><a href="https://github.com/auth0/go-auth0/compare/v0.10.1...v0.11.0">Full Changelog</a></p>
<h3>⚠️ Notice of Breaking Changes</h3>
<p>Please note that this version introduces many minor breaking changes. The intention with this release was to capture as many breaking changes into a single release to minimize future disruption.</p>
<p>These changes were introduced for three primary reasons:</p>
<ul>
<li>Decoupling from <a href="https://github.com/auth0/terraform-provider-auth0">Auth0 Terraform Provider</a> implementation</li>
<li>More precise typing of struct properties for better DX</li>
<li>Enabling Auth0 Terraform provider to explicitly set empty values (see: <a href="https://github-redirect.dependabot.com/auth0/terraform-provider-auth0/issues/14">related Github issue</a>)</li>
</ul>
<p>Review the complete list below before upgrading.</p>
<h3>Changed</h3>
<ul>
<li><code>Action</code> struct fields
<ul>
<li><code>SupportedTriggers</code> to <code>[]ActionTrigger</code> type</li>
<li><code>Dependencies</code> to <code> *[]ActionDependency</code> type</li>
<li><code>Secrets</code> to <code>*[]ActionSecret</code> type</li>
</ul>
</li>
<li><code>Client</code> struct fields
<ul>
<li><code>Callbacks</code> to <code>*[]string</code> type</li>
<li><code>AllowedOrigins</code> to <code>*[]string</code> type</li>
<li><code>WebOrigins</code> to <code>*[]string</code> type</li>
<li><code>ClientAliases</code> to <code>*[]string</code> type</li>
<li><code>AllowedClients</code> to <code>*[]string</code> type</li>
<li><code>AllowedLogoutURLs</code> to <code>*[]string</code> type</li>
<li><code>EncryptionKey</code> to <code>*map[string]string</code> type</li>
<li><code>GrantTypes</code> to <code>*[]string</code> type</li>
<li><code>ClientMetadata</code> to <code>*map[string]string</code> type</li>
<li><code>Mobile</code> to dedicated <code>*ClientMobile</code> type</li>
<li><code>Scopes</code> to <code>*map[string]string</code> type</li>
</ul>
</li>
<li><code>ClientNativeSocialLogin</code> struct
<ul>
<li><code>Apple</code> to dedicated <code>*ClientNativeSocialLoginSupportEnabled</code> type</li>
<li><code>Facebook</code> to dedicated <code>*ClientNativeSocialLoginSupportEnabled</code> type</li>
</ul>
</li>
<li><code>ClientGrant</code> struct field
<ul>
<li><code>Scope</code> to <code>[]string</code> type</li>
</ul>
</li>
<li><code>Connection</code> struct fields
<ul>
<li><code>EnabledClients</code> to <code>*[]string</code> type</li>
<li><code>Realms</code> to <code>*[]string</code> type</li>
<li><code>Metadata</code> to <code>*map[string]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptions</code> struct fields</li>
<li><code>CustomScripts</code> to <code>*map[string]string</code> type</li>
<li><code>Configuration</code> to <code>*map[string]string</code> type</li>
<li><code>ConnectionOptionsGoogleOAuth2</code> struct field
<ul>
<li><code>AllowedAudiences</code> to <code>*[]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptionsOIDC</code> struct field
<ul>
<li><code>DomainAliases</code> to <code>*[]string</code> type</li>
</ul>
</li>
<li><code>ConnectionOptionsOAuth2</code> struct field</li>
</ul>

</blockquote>
<p>... (truncated)</p>
</details>
<details>
<summary>Commits</summary>
<ul>
<li><a href="https://github.com/auth0/go-auth0/commit/9f48e05aa94ff3e4a93c8d700f7d046b80509b86"><code>9f48e05</code></a> v0.11.0</li>
<li><a href="https://github.com/auth0/go-auth0/commit/12d8a914be750ce18ff43863a0528557777698f0"><code>12d8a91</code></a> Updating changelog for 0.11.0 (<a href="https://github-redirect.dependabot.com/auth0/go-auth0/issues/117">#117</a>)</li>
<li><a href="https://github.com/auth0/go-auth0/commit/1e024abc8558c285eecb5300f5f235aa4ee7d82d"><code>1e024ab</code></a> DXCDT-218: Allow empty fields through pointers  (<a href="https://github-redirect.dependabot.com/auth0/go-auth0/issues/113">#113</a>)</li>
<li>See full diff in <a href="https://github.com/auth0/go-auth0/compare/v0.10.1...v0.11.0">compare view</a></li>
</ul>
</details>
<br />

### CODE REVIEW

File: internal/cli/utils_shared.go [hunk 1]
```diff
@@ -55,7 +53,7 @@ func BuildOauthTokenParams(clientID, clientSecret, audience string) url.Values {
 }
 
 // runClientCredentialsFlow runs an M2M client credentials flow without opening a browser
-func runClientCredentialsFlow(cli *cli, c *management.Client, clientID string, audience string, tenant Tenant) (*authutil.TokenResponse, error) {
+func runClientCredentialsFlow(_ *cli, c *management.Client, clientID string, audience string, tenant Tenant) (*authutil.TokenResponse, error) {
```
Comment (Widcket) [inline internal/cli/utils_shared.go:56]: Maybe we can remove the `_ *cli` parameter?

File: internal/display/organizations.go [hunk 2]
```diff
@@ -78,17 +78,17 @@ func (r *Renderer) OrganizationUpdate(organization *management.Organization) {
 	r.Result(makeOrganizationView(organization, r.MessageWriter))
 }
 
-func makeOrganizationView(organization *management.Organization, w io.Writer) *organizationView {
+func makeOrganizationView(organization *management.Organization, _ io.Writer) *organizationView {
```
Comment (Widcket) [inline internal/display/organizations.go:81]: Can we remove the `_ io.Writer` parameter?

File: internal/cli/utils_shared.go [hunk 3]
```diff
@@ -28,12 +28,10 @@ const (
...
 )
 
-var (
-	cliLoginTestingScopes []string = []string{"openid", "profile"}
-)
+var cliLoginTestingScopes = []string{"openid", "profile"}
 
 func BuildOauthTokenURL(domain string) string {
-	var path string = "/oauth/token"
+	var path = "/oauth/token"
```
Comment (Widcket) [inline internal/cli/utils_shared.go:34]: 👏🏼

File: internal/cli/utils_shared.go [hunk 4]
```diff
@@ -28,12 +28,10 @@ const (
 	manageURL                        string = "https://manage.auth0.com"
 )
 
-var (
-	cliLoginTestingScopes []string = []string{"openid", "profile"}
-)
+var cliLoginTestingScopes = []string{"openid", "profile"}
```
Comment (Widcket) [inline internal/cli/utils_shared.go:31]: 👏🏼

File: internal/display/organizations.go (hunk 2 above)
Comment (sergiught) [reply internal/display/organizations.go]: Fixed in https://github.com/auth0/auth0-cli/pull/489/commits/802fb15094837e7be3b28f8866430d2f9d5f887f

File: internal/cli/utils_shared.go (hunk 1 above)
Comment (sergiught) [reply internal/cli/utils_shared.go]: Fixed in https://github.com/auth0/auth0-cli/pull/489/commits/802fb15094837e7be3b28f8866430d2f9d5f887f

**Review by willvedd — APPROVED** [review]
Reviewed together with @sergiught 👍

**Pull request merged**